packages/jest-mock-apollo/src/factory.ts

```diff
@@ -0,0 +1,81 @@
+import {buildSchema} from 'graphql';
+import {GraphQLRequest, ApolloLink} from 'apollo-link';
+import {
+  InMemoryCache,
+  IntrospectionFragmentMatcher,
+} from 'apollo-cache-inmemory';
+import ApolloClient from 'apollo-client';
+import {GraphQLMock, GraphQLClientConfig, GraphQLClientOptions} from './types';
+import MockApolloLink from './MockApolloLink';
+import Requests from './Requests';
+
+export type MockGraphQLClient = ApolloClient<any> & {
```

Comment:
For future discussion, but should we just have a custom GraphQL client class that has the additional fields?